e2e/provision/playbooks/roles/install/tasks/main.yml

```diff
@@ -13,7 +13,7 @@
     name: kpt
   loop:
     - {pkg: porch-dev, version: porch-dev/v2, namespaces: [porch-fn-system, porch-system]}
-    - {pkg: nephio-controllers, version: nephio-controllers/v4, namespaces: [nephio-system]}
+    - {pkg: nephio-controllers, version: nephio-controllers/v5, namespaces: [nephio-system]}
```

Comment:
Is there a way to pull this version info instead of hardcoding it here? It can be done in a later PR hopefully

Reply:
Well, if we do that then we don't have a repeatable deployment. But agreed, this needs rework in R2 or R1++ to be simpler.

What I *can* do for now is force-push tags to move v5 to a new commit if I need another change. That doesn't go through our review process...